### PR TITLE
docs: upgrade the documentation for PlatformAdmin

### PR DESCRIPTION
**Pull request by Pluviophile225**
**What type of PR is this?**
changes about docs
/user-manuals/iot/edgex-foundry.md

**What this PR does / why we need it:**
modify iot/edgex-foundry.md

**Which issue(s) this PR fixes:**
Fixes  [#1579](https://github.com/openyurtio/openyurt/issues/1579)

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/iot/edgex-foundry.md [hunk 1]
```diff
@@ -13,276 +13,100 @@ title: 云原生管理端设备
 
 # 环境要求
 
-- OpenYurt v0.5.0+
-- 安装了 [Yurt-app-manager](https://github.com/openyurtio/yurt-app-manager) 组件
+- OpenYurt v1.4.0+
+- 安装了 [Yurt-Manager](https://github.com/openyurtio/yurt-app-manager) 组件
```
Comment (LavenderQAQ) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/iot/edgex-foundry.md:17]: `yurt-app-manager` has been archived, you could remove this link.

File: i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/iot/edgex-foundry.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: 云原生管理端设备
 ---
 
 
-本文档主要讲述如何在已有的OpenYurt集群上安装Yurt-Device-Controller 和 Yurt-EdgeX-Manager组件，并通过部署虚拟端设备来展示如何通过云原生的方式管理边缘端设备。
+本文档主要讲述如何在已有的OpenYurt集群上安装YurtIoTDock，并通过部署虚拟端设备来展示如何通过云原生的方式管理边缘端设备。
 
 对于有兴趣的读者，可以去相关的github仓库参考本文使用组件的具体实现：[Yurt-Device-Controller](https://github.com/openyurtio/yurt-device-controller)
```
Comment (LavenderQAQ) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/iot/edgex-foundry.md:8]: Just to add to the current situation: `yurt-edgex-manager` and `yurt-device-controller` have been merged into the main repo, and then introduce the new crd and controller.

**Review by LavenderQAQ — CHANGES_REQUESTED** [review]

File: i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/iot/edgex-foundry.md [hunk 3]
```diff
@@ -13,276 +13,100 @@ title: 云原生管理端设备
 
 # 环境要求
```
Comment (LavenderQAQ) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/iot/edgex-foundry.md:14]: It needs to be changed to "##".

File: i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/iot/edgex-foundry.md [hunk 4]
```diff
@@ -13,276 +13,100 @@ title: 云原生管理端设备
...
 - 将访问coreDNS service的流量改为节点池内闭环，参考[教程](https://github.com/openyurtio/openyurt/blob/master/docs/tutorial/service-topology.md)
 
 
 
-# 1. 安装yurt-edgex-manager并创建一个EdgeX实例
-
-安装部署yurt-edgex-manager
-
-
+# 1. 创建节点池
```
Comment (LavenderQAQ) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/iot/edgex-foundry.md:24]: It needs to be changed to "##".

File: i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/iot/edgex-foundry.md [hunk 5]
```diff
@@ -13,276 +13,100 @@ title: 云原生管理端设备
...
+NAME       TYPE    READYNODES   NOTREADYNODES   AGE
+beijing    Cloud   1            0               4m22s
+shanghai   Edge    1            0               3m39s
 ```
 
-# 2. 安装并部署yurt-device-controller
-
-安装yurt-device-controller 相关的CRD
-
+# 2. 在节点池内创建IoT系统PlatformAdmin
```
Comment (LavenderQAQ) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/iot/edgex-foundry.md:80]: It needs to be changed to "##".

File: i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/iot/edgex-foundry.md [hunk 6]
```diff
@@ -3,7 +3,7 @@ title: 云原生管理端设备
 ---
 
 
-本文档主要讲述如何在已有的OpenYurt集群上安装Yurt-Device-Controller 和 Yurt-EdgeX-Manager组件，并通过部署虚拟端设备来展示如何通过云原生的方式管理边缘端设备。
+本文档主要讲述如何在已有的OpenYurt集群上安装YurtIoTDock，并通过部署虚拟端设备来展示如何通过云原生的方式管理边缘端设备。
```
Comment (LavenderQAQ) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/iot/edgex-foundry.md:6]: "本文档主要讲述如何在已有的OpenYurt集群上使用PlatformAdmin部署EdgeX系统和YurtIoTDock"

File: i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/iot/edgex-foundry.md (hunk 2 above)
Comment (LavenderQAQ) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/iot/edgex-foundry.md:8]: 在 OpenYurt v1.4 版本中，我们升级了原来的 yurt-edgex-manager 和 yurt-device-controller 组件。前者已经内置于 yurt-manager 中，用户可以通过编写 Yaml 文件来创建 PlatformAdmin 资源，简单几行配置就可以往节点池内构建一套完整的 EdgeX 系统；后者更名为 yurt-iot-dock ，会在 PlatformAdmin 创建时自动下发到边缘侧，实现帮助用户一键实现端设备托管的能力。

**Review by LavenderQAQ — CHANGES_REQUESTED** [review]

**Comment by rambohe-ch** [comment]
@Pluviophile225 how about update the english version of this tutorial?

**Review by LavenderQAQ — APPROVED** [review]
/lgtm

**Review by LavenderQAQ — CHANGES_REQUESTED** [review]
I will complete the platformadmin documentation.

**Review by LavenderQAQ — APPROVED** [review]
This pr is ready.

**Pull request merged**